### PR TITLE
Try glue2

### PR DESCRIPTION
**Pull request by crecine**
### Summary

Follow on to #397 

### Related Issues

- Resolves #

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

File: aviary/docs/getting_started/input_csv_phase_info.ipynb [hunk 1]
```diff
@@ -18,15 +47,8 @@
...
     "A portion of an example vehicle .csv file is shown below:\n",
     "\n",
-    "```\n",
-    "aircraft:avionics:mass_scaler,1.2,unitless\n",
-    "aircraft:crew_and_payload:baggage_mass_per_passenger,45.0,lbm\n",
-    "aircraft:crew_and_payload:mass_per_passenger,180.0,lbm\n",
-    "aircraft:wing:span,117.83,ft\n",
-    "aircraft:wing:strut_bracing_factor,0.0,unitless\n",
-    "aircraft:wing:surface_ctrl_mass_scaler,1.0,unitless\n",
-    "aircraft:wing:sweep,25.0,deg\n",
```
Comment (crecine) [inline aviary/docs/getting_started/input_csv_phase_info.ipynb:28]: now this is:
```
aircraft:air_conditioning:mass_scaler,1.0,unitless
aircraft:canard:area,0.0,ft**2
aircraft:crew_and_payload:cargo_container_mass_scaler,1.0,unitless
aircraft:crew_and_payload:non_flight_crew_mass_scaler,1.0,unitless
mission:design:range,3500,NM
mission:summary:fuel_flow_scaler,1.0,unitless
settings:equations_of_motion,height_energy
```

File: aviary/docs/examples/modified_aircraft.csv
Comment (ehariton) [inline aviary/docs/examples/modified_aircraft.csv:1]: Why is this file included in the commit? Was this an accidental include?

Comment (ehariton) [reply aviary/docs/examples/modified_aircraft.csv:1]: Same question for aviary/docs/examples/outputted_phase_info.py

Comment (crecine) [reply aviary/docs/examples/modified_aircraft.csv:1]: It was expected by aviary/docs/examples/more_advanced_example.ipynb and aviary/docs/user_guide/drawing_and_running_simple_missions.ipynb

**Review by ehariton — APPROVED** [review]

**Review by Kenneth-T-Moore — APPROVED** [review]
